### ISSUE TITLE
Add an email address for a specific matter

### ISSUE THREAD

**Issue by signed-log** [datenanfragen/data#2135]
What is the most optimal way to add an email address that is only to be used for a specific matter?

A transport company has a `retraitvideo@semitan.fr` address. However, it is clearly stated on the website that all requests sent to that address **must** be related to security cameras matters, and not general matters, which are to be sent through the web form or through the mail

I was eying putting it into a comment, but I don't know what is the preferred way to deal with this is

**Comment by mal-tee** [comment]
I don't think we had this case before. :D

I see two options:

1. As a comment, yes
2. Or duplicate the record with different email addresses
	- and denote in `runs` that one is only for security cameras
	
But I think option two creates two much overhead and might confuse users; Imo option 1 is the best for the time being. What do you think @baltpeter ?

**Comment by baltpeter** [comment]
I agree with @mal-tee. A comment would be the only sensible option here.

In the long run, I'm leaning towards entirely deprecating comments, and moving all relevant existing comments to machine-readable data. But to be honest, I don't think we would handle this case then. Out of 2500+ companies in the database, less than a handful have multiple email addresses for different purposes…

**Comment by mal-tee** [comment]
> I don't think we would handle this case then.

That's why I oppose of getting rid of comments completely. :D

**Comment by baltpeter** [comment]
> That's why I oppose of getting rid of comments completely. :D

But comments are always only an ugly hack with a horrible user experience. In this example, we have the problem of the translations but even worse yet is how you would actually use that. Currently, comments aren't displayed at all in the new generator. But let's imagine they were. You may or may not notice the comment. Then, you generate your request, open the modal but you realize "oh wait, there was a different email for my use case), so you close the modal, go back to the comment, copy the email address, re-open the modal, and then you manually have to change the email.

If there is some information we think should be part of a record, we should add a machine-readable field and implement proper support for that. Everything else can go into a comment-comment (the terminology is awful as well :D).
